workshops/templatetags/navigation.py

```diff
@@ -12,23 +34,38 @@ def navbar_element(context, title, url_name):
     is later reversed into proper URL.
     """
     url = reverse(url_name)
-    return navbar_element_url(context, title, url)
+    active = context['request'].path == url
```

Comment:
Yes, it does. If the `url` and the `request.path` are the same, then the navigation element should be highlighted as active.